script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
+    
+    if (loadedTODO !== null) {
+        const parsedTODO = JSON.parse(loadedTODO); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedTODO.forEach(function(input) {
+            printData(input.text,parseInt(input.id),true)
+        });
+    }
+    if (loadedFIN !== null) {
+        const parsedFIN = JSON.parse(loadedFIN); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedFIN.forEach(function(input) {
+            printData(input.text,parseInt(input.id),false)
+        });
+    }
+}
+
+function saveDATA(){
+    localStorage.setItem(TODODATA, JSON.stringify(toDoArr));
+    localStorage.setItem(FINDATA, JSON.stringify(finArr));
+}
+
+function setLength(){
+    lenTodo.innerText = toDoArr.length
+    lenFin.innerText = finArr.length
+}
+
+function deleteData(arr,target){
+    // filter target from arr
+    // target.id가 string으로 가져와짐 - 그래서 처음에 값을 비교를 못했음
+    const result = arr.filter(element => element.id!==target.id)
+    return result
+}
+
+function deleteHandler(event){
+    // event.target의 class를 보고 todo 인지 finished인지 판별 가능
+    const target = event.target.parentNode.parentNode
+    if(target.className === "todo"){
+        // delete from todoArr
+        toDoArr = toDoArr.filter(element => element.id !== target.id)
+        // remove from parent
+        toDoList.removeChild(target);
+    }else{
+        // delete from finArr
+        finArr = finArr.filter(element => element.id !== target.id)
+        finishedList.removeChild(target);
+    }
+    setLength();
+    console.log(toDoArr);
+    console.log(finArr);
+    saveDATA();
+}
+
+function moveData(event){
+    const target = event.target.parentNode
+    if(target.className === "todo"){
+        //move to bottom
+        target.classList.remove("todo");
+        target.classList.add("fin");
+        finArr = [...finArr,{id:target.id,text:target.firstChild.innerHTML}]
```

Comment:
`spread operator`를 써서 간편하게 `finArr`를 업데이트 하신 부분이 인상적이네요!

다만 `text`를 조금 더 명확히 알 수 있도록 선언해 주시면 더 좋을 것 같습니다!
```suggestion
        const newFinishedToDoContent = target.firshChild.innerHTML
        finArr = [...finArr,{id:target.id,text: newFinishedToDoContent}]
```